backend/src/RedisDriver.js

```diff
@@ -0,0 +1,29 @@
+const Redis = require( 'ioredis' );
+const redisAdapter = require( 'socket.io-redis' );
+const dotenv = require( 'dotenv' );
+dotenv.config( { path: '.env' } );
+
+class RedisDriver {
+	constructor( io ) {
+		this.io = io;
+		this._config = {
+			host: process.env.REDIS_HOST,
```

Comment:
I think it is a better practice to inject config by the constructor.

Reply:
Done.